DQM/SiStripMonitorApproximateCluster/python/SiStripMonitorApproximateCluster_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+# SiStripMonitorCluster
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+SiStripMonitorApproximateCluster = DQMEDAnalyzer("SiStripMonitorApproximateCluster",
```

Comment:
Why not rely on the generated `cfi` file?

Reply:
Addressed at https://github.com/cms-sw/cmssw/pull/40295/commits/a90f8e55543f72001dfd4eafd3a64d2a4b362cd5